backend/update_requirements.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -euo pipefail
+
+exec docker run -v "$PWD:/src" -w /src --rm python:3.5 bash \
```

Comment:
Thank you!

Can you duplicate this script in `sample-test` folder too?
Very useful!

Reply:
Hrm it can be invoked from `sample-test`, which is mentioned in the README I've created there. Maybe we should move it to the root of the repo? Or in `/bin` or `/hack` or something?